source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,218 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
+
+Before reading this section, you should be familiar with the
+[__Making payments__ section](/payment_flow_overview/#payment-flow).
+
+If you have already [integrated with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api), you
+can start using Direct Debit quickly. You should:
+
+* request a GOV.UK Pay Direct Debit test account [from our
+support team](/support_contact_and_more_information/#contact-us).
+* add support for the Direct Debit endpoints to your existing GOV.UK Pay
+integration
+* sign in to your GOV.UK Pay Direct Debit account and follow the instructions
+to set up an account with our Direct Debit supplier
+
+## GOV.UK Pay ‘agreements’
+
+In order to support Direct Debit, GOV.UK Pay now uses ‘agreements’. An
+agreement represents an individual payer’s consent for future payments using a
+particular funding source.
+
+For Direct Debit transactions, GOV.UK Pay fulfils an agreement by registering
+a Direct Debit mandate with the payer’s bank. 
+
+## Synchronous and asynchronous payment methods
```

Comment:
it's probably in there to explain why we've called it agreement, rather than mandate. but it's a little abstruse, if that was the intention...

Reply:
I'm not sure if I follow. It might be me having made edits to the original content and forgetting, honestly (if so, apologies), or my lack of understanding, but as things stand I don't see how this reinforces "agreement" vs "mandate".